helm-charts/falcon-image-analyzer/templates/daemonset.yaml

```
@@ -1,6 +1,38 @@
 {{- if .Values.daemonset.enabled }}
 {{- $imagePullSecret := include "falcon-image-analyzer.imagePullSecret" . }}
 {{- $registryConfigJson := include "falcon-image-analyzer.registryConfigJson" . }}
+{{- $name := (printf "%s" (include "falcon-image-analyzer.iarAgentService" .)) -}}
+{{- $fullName := (printf "%s.%s.svc" $name .Release.Namespace) -}}
```

We will need to update all references to `.Release.Namespace` to something like:

```suggestion
{{- $iarNamespace := include "falcon-image-analyzer.namespace" . }}
{{- $fullName := (printf "%s.%s.svc" $name $iarNamespace) -}}
```

And then reuse that variable in this template, rather than using `.Release.Namespace` directly.

---

Lets do the clean up post this release . As this was tested as is